README.md

```diff
@@ -25,10 +39,14 @@ nix = "0.6.0"
 
 Then, add this to your crate root:
 
-```rust
+```rust,ignore
```

Comment:
Out of curiosity: what does the `ignore` 'language' achieve? A cursory search didn't bring up anything useful.